### PR TITLE
Fix spelling mistakes throughout CCIE documentation

### PR DESCRIPTION
**Pull request by wgraysen**
Similar to #1127 and #1132, I'm reading through all of the CircleCI documentation as part of my training. While I was reading through the CCIE documentation, I found a few spelling mistakes and a couple of other things that I thought could be improved. See the commit history/diff for which specific words were corrected and in what files. 🧀

### CODE REVIEW

File: jekyll/_ccie/gpu-configuration.md [hunk 1]
```diff
@@ -206,4 +206,4 @@ Then in your circle.yml test section, you'll need to add
     - export LD_LIBRARY_PATH=/usr/local/cuda-8.0/lib64:$LD_LIBRARY_PATH
 
     
-Now you'll have GPU access in your products!
\ No newline at end of file
+Now you'll have GPU access in your products!
```
Comment (felicianotech) [inline jekyll/_ccie/gpu-configuration.md]: GitHub is highlighting this line as being changed but I don't see what the change is.

Comment (wgraysen) [inline jekyll/_ccie/gpu-configuration.md]: I have no idea. That's really strange. I didn't change anything on that line and can't see any differences in the diff either.

Comment (wgraysen) [inline jekyll/_ccie/gpu-configuration.md]: 🤔

Comment (felicianotech) [inline jekyll/_ccie/gpu-configuration.md]: I looked it up in Git. The file didn't end in a newline character (a `\n`) which is a POSIX/Linux standard. It was added by the text editor. That character doesn't show up. All is good. :)

Comment (wgraysen) [inline jekyll/_ccie/gpu-configuration.md]: Ah, okay. I was editing the file with Xcode and it sometimes does that.
Thanks for clearing that up! 😄

**Comment by wgraysen** [comment]
Since these are just spelling mistake fixes and whatnot, I'm going to merge since I got @felicianotech's ✅ .

**Pull request merged**